src/EnrollWebauthnController.js

```diff
@@ -79,7 +79,12 @@ function (Okta, Errors, FormType, FormController, CryptoUtil, webauthn, Footer,
               },
               excludeCredentials: getExcludeCredentials(activation.excludeCredentials)
             });
-            return new Q(navigator.credentials.create({publicKey: options}))
+            var webauthnAbortController = new AbortController();
```

Comment:
`AbortController` seems not compatible in older browser. would there be any regression concern?
https://www.caniuse.com/#feat=abortcontroller

Reply:
@haishengwu-okta  older meaning IE 11 ? webauthn  itself wont work so we dont have to worry about it I guess. ie) we will safely fail before we use the abortController.